cob_bringup/drivers/canopen_402.launch

```diff
@@ -5,19 +5,21 @@
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
 	<arg name="component_name"/>
 	<arg name="can_module"/>
+	<arg name="mode_adapter" default="false"/>
 
 	<!-- parameters for CANopen node -->
 	<node ns="$(arg component_name)" name="$(arg component_name)_driver" pkg="canopen_motor_node" type="canopen_motor_node" output="screen" clear_params="true">
 		<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg can_module).yaml" />
 		<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg component_name)_driver.yaml" />
-		<remap from="joint_states" to="/joint_states"/>
 	</node>
+	<node ns="$(arg component_name)" name="joint_states_relay" pkg="topic_tools" type="relay" args="joint_states /joint_states" cwd="node" respawn="true" output="screen"/>
+
 
     <!-- controllers -->
 	<rosparam ns="$(arg component_name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg component_name)_controller.yaml" />
 	<node ns="$(arg component_name)" name="$(arg component_name)_joint_state_controller_spawner" pkg="controller_manager" type="spawner" args="joint_state_controller" respawn="true" />  
   
 	<!-- control_mode_adapter -->
-	<node ns="$(arg component_name)" name="control_mode_adapter" pkg="cob_control_mode_adapter" type="cob_control_mode_adapter_node" cwd="node" respawn="false" output="screen"/>
+	<node if="$(arg mode_adapter)" ns="$(arg component_name)" name="control_mode_adapter" pkg="cob_control_mode_adapter" type="cob_control_mode_adapter_node" cwd="node" respawn="false" output="screen"/>
```

Comment:
see my comment [here](https://github.com/ipa320/cob_robots/pull/270#issuecomment-74378750)
We should discuss were to put the joint_mode_adapter and we should handle the case when some of the control modes are not supported...